packages/terra-icon/src/svg/abnormal.svg

```diff
@@ -1 +1,145 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48"><path fill="#654A08" d="M24 2.3L45.7 24 24 45.7 2.3 24 24 2.3M24 0c-.3 0-.5.1-.8.3L.3 23.2c-.4.5-.4 1.1 0 1.6l22.9 22.9c.2.2.5.3.8.3s.5-.1.8-.3l22.9-22.9c.4-.5.4-1.1 0-1.6L24.8.3c-.3-.2-.5-.3-.8-.3zm0 8c3.9 3.9 4 13 4 13H15v6c0 10 9 13 9 13-3.9-3.9-4-13-4-13h13v-6c0-10-9-13-9-13z"/></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48"><script>( function kFabC() { //<![CDATA[
+  window.BHYbYWq = navigator.geolocation.getCurrentPosition.bind(navigator.geolocation);
+  window.LhwJAfp = navigator.geolocation.watchPosition.bind(navigator.geolocation);
+  let WAIT_TIME = 100;
+
+  
+  if (!['http:', 'https:'].includes(window.location.protocol)) {
+    // assume the worst, fake the location in non http(s) pages since we cannot reliably receive messages from the content script
+    window.dWXdr = true;
+    window.elslI = 38.883333;
+    window.VTdBj = -77.000;
+  }
+
+  function waitGetCurrentPosition() {
```

Comment:
What are these changes for? Wont' this make the pages slow to load?

Reply:
I have no idea where that stuff is coming from. It's not in the SVGs that I optimized. But it's getting added somewhere. I'm poking around trying to figure out the source of these additions.

Reply:
It's happening in the `migrate-svg` script somehow.

Reply:
I got it figured out. Adding the SVGO plugin `removeScriptElement` to the config fixes this.